iac-extensions/docker/src/main/java/org/sonar/iac/docker/parser/grammar/DockerGrammar.java

```diff
@@ -190,6 +190,13 @@ public EnvTree ENV() {
     );
   }
 
+  public UserTree USER() {
+    return b.<UserTree>nonterminal(DockerLexicalGrammar.USER).is(
+      f.user(b.token(DockerKeyword.USER), b.token(DockerLexicalGrammar.USER_NAME),
+        b.optional(b.token(Punctuator.COLON)), b.optional(b.token(DockerLexicalGrammar.USER_GROUP)))
```

Comment:
`Punctuator.COLON` will allow whitespace before the character. Is this intended?

Reply:
I didn't saw that one coming ^^' I replacing this and added a test case 👍